### DISCUSSION TITLE
E: Package 'libvisp-dev' has no installation candidate

### DISCUSSION THREAD

**Discussion by jimingda** [lagadic/visp#1284] [Q&A]
Hi,

My system is `Ubuntu 20.04.4 LTS`, when i installed `visp` via `sudo apt-get install libvisp-dev libvisp-doc visp-images-data`, i got error as below.  Anyone has the similar problem or how to fix it?

Thanks in advance.
```
icub@icubsrv:~$ sudo apt-get install libvisp-dev libvisp-doc visp-images-data
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package libvisp-dev is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'libvisp-dev' has no installation candidate
E: Unable to locate package libvisp-doc
```

**Comment by fspindle** [comment]
Unfortunately visp is not packaged for Ubuntu 20.04.
It is for 18.04, 22.04 and should be available for 24.04

**Reply by jimingda** [reply]
Thanks @fspindle 

I will try to install via source-code.
